backend/src/models/Application.ts

```diff
@@ -1,14 +1,28 @@
 import { InferSchemaType, Schema, model } from "mongoose";
 
+export enum ApplicationStatus {
+  APPLIED = "APPLIED",
+  OA = "OA",
+  PHONE = "PHONE",
+  FINAL = "FINAL",
+  OFFER = "OFFER",
+  REJECTED = "REJECTED",
+}
```

Comment:
Could you name this `Status` instead of `ApplicationStatus` to avoid confusion and can you disable the lint warning for the enum:
```
/* eslint-disable no-unused-vars */
export enum Status {
 ...
}
/* eslint-enable no-unused-vars */
```